### DISCUSSION TITLE
Need Help: Plots & Graphs: What Am I Missing

### DISCUSSION THREAD

**Discussion by AzamBukhari** [AammarTufail/python_ka_chilla2023#56] [Q&A] [answered]
Hello & AoAlaikum

Any idea what I am missing here
My code runs successfully. but graphs are not coming up.

![image](https://user-images.githubusercontent.com/121688289/211889321-57e7240d-ca34-4131-8452-6df4252ff6d5.png)

**Comment by nouman073** [comment, marked as answer]
%matplotlib inline 
run this line code before ploting

**Reply by AzamBukhari** [reply]
Thank you Janab
